lib/solarwinds_apm/api/transaction_name.rb

```diff
@@ -38,9 +38,13 @@ module TransactionName
       # * Boolean
       #
       def set_transaction_name(custom_name = nil)
+        otel_config = SolarWindsAPM::OTelNativeConfig if defined? SolarWindsAPM::OTelNativeConfig
+        otel_config = SolarWindsAPM::OTelConfig if defined? SolarWindsAPM::OTelConfig
```

Comment:
Do we need both of these assignments?

Reply:
No, I will remove the `SolarWindsAPM::OTelConfig if defined? SolarWindsAPM::OTelConfig` once merge everything together and make the final version in branch `pure-otel-ruby`